### ISSUE TITLE
Testcafe not able to launch Chrome browser

### ISSUE THREAD

**Issue by vikaschoudhary89** [DevExpress/testcafe#7021]
### What is your Scenario?

I am running my test in multiple browsers, working fine for all other browser except Chrome as testcafe is not able to launch chrome browser.

### What is the Current behavior?

Chrome browser is not getting launched by testcafe

### What is the Expected behavior?

Testcafe should be able to start/launch chrome browser.

### What is your public website URL? (or attach your complete example)

NA as happening for all URLs

### What is your TestCafe test code?

// not an issue with test so just giving reference example from testcafe website

import { Selector } from 'testcafe';

const checkbox = Selector('#tried-test-cafe');

fixture`TestController.click`
    .page`https://devexpress.github.io/testcafe/example/`;

test('Click a check box and check its state', async t => {
    await t
        .click(checkbox)
        .expect(checkbox.checked).ok();
});

### Your complete configuration file

{
    "browsers": ["chrome"],
    "src": ["tests/loginpagetest.ts"],
    "screenshots": {
        "takeOnFails": true,
        "path": "artifacts/screenshots",
        "fullPage": true,

        "pathPattern": "${USERAGENT}_${DATE}_${TIME}/${FIXTURE}/${TEST}/${TEST_INDEX}/${FILE_INDEX}.png"
        
    },
    
    "debugOnFail": false,
    "skipJsErrors": true,
    "concurrency": 1,
    "speed": 1,
    "proxy": "XXXX.XXX.XXX.XX:1XX0",
    "browserInitTimeout": 300000

}

### Your complete test report

ERROR Cannot establish one or more browser connections.
1 of 1 browser connections have not been established:
- chrome

Hints:
- Increase the value of the "browserInitTimeout" option if it is too low (currently: 5 minutes for all browsers). This option determines how long TestCafe waits for browsers to be ready.
- The error can also be caused by network issues or remote device failure. Make sure that your network connection is stable and you can reach the remote device.

### Screenshots

![Uploading image.png…]()

### Steps to Reproduce

1. Run the test in Chrome browser , **_ testcafe chrome .\tests\firsttest.ts _**
2.
3.

### TestCafe version

1.18.4

### Node.js version

v16.15.0

### Command-line arguments

testcafe chrome .\tests\firsttest.ts

### Browser name(s) and version(s)

Chrome 101.0.4951.64 

### Platform(s) and version(s)

Windows 10

**Comment by Aleksey28** [comment]
Hi @vikaschoudhary89,

I cannot reproduce the issue using this example. Please give me some information.

1. Do you run tests on your local machine?
2. Is the issue reproduced without proxy?

**Comment by vikaschoudhary89** [comment]
Thanks @Aleksey28  for your response. Apologies for the delay from my end.

1. Yes, I am running in my local machine.
2. Yes, tried with and without proxy and same issue is happening. 

![image](https://user-images.githubusercontent.com/53804006/170948409-3927c8dc-63d8-43d6-b023-87cbdeae5878.png)

**Comment by Aleksey28** [comment]
I didn't manage to reproduce this issue with exactly this test and config without a proxy.
Testcafe: 1.18.4
Node: 16.15.0
Chrome: 102.0.5005.61
OS: Windows 10

Please check again your versions of Testcafe, Node, and Chrome. In addition, check that the error occurs exactly with this test.

**Comment by vikaschoudhary89** [comment]
Hi @Aleksey28 ,
yes the versions are correct, I tried in old and new version of testcafe and Node.

Actually this test used to run in my old machine with same configuration but I got new laptop from my organization and same test in same config stopped working (works fine in FF and Edge).

I am not getting any error in test, as browser itself is not getting launched but same tests are running in FF and Edge successfully.
